src/Commands/ProjectCreateCommand.php

```diff
@@ -463,7 +466,8 @@ function ($state) use ($ci_env, $siteDir, $use_ssh) {
             ->progressMessage('Beginning CI testing')
             ->taskCIStartTesting()
                 ->provider($this->ci_provider)
-                ->environment($ci_env);
+                ->environment($ci_env)
+                ->setCluCronSchedule($cluCronPattern);
```

Comment:
The CLU cron schedule pattern is exactly the sort of configuration data that the $ci_env was intended to store. The parameter list for `startTesting` et. al. would get quite complicated if we had to list every configuration value as a parameter in the call stack. If you set a value on $ci_env here, then you should be able to fetch it by its key in startTesting, where it is needed.

Reply:
ah, great, thanks for the feedback.
i'll look into putting the param into ci_env instead.